### ISSUE TITLE
Lycanites Mobs Discussion/Balance

### ISSUE THREAD

**Issue by DrasticDemise** [DrasticDemiseFTBPacks/Banished#44]
I need some more information on Lycanites as I simply don't have enough time to test every single mob. I need to know:
- What mobs have you run into that are impossible to kill early-mid game?
- What mobs have an unusual drop rate that could be changed?
- Have you found a mob that drops string? How common are they?
- Have the vanilla mobs suddenly stopped spawning or had their spawn rate drastically reduced? (This has been buffed for next version)
- Post mobs below that you would like to see a dark power quest for.

**Comment by Mechalith** [comment]
I've seen very few vanilla mobs post-update, it seems like Lycanite's might be flooding them out. The inferno mobs in specific are probably too common, esp in concert with the lava monsters, given the gigantic lava lake right near the portal. Might want to disable the lava fish-lizard thing that I can't remember the name of.

Only other thing that comes to mind is that the shadow creatures are especially cruel when you've only got 32 torches and no daylight to work with. I forgot they existed at first and nearly got my head eaten by a grue 20 sec after I came out of the portal b/c I chugged the nightvision potion and headed for my usual housing spot without bothering to light things as I went.

Setting up dark power quests for the Lycanite mobs will help a lot. The torches wouldn't have been such an issue, but without any vanilla mobs spawning and no way to leave my light bubble, getting enough power to buy the magelight spell or plant a tree for torches took some doing. (I ended up breaking a node and using the essences out of desperation.)

Haven't found any mobs that drop string other than spiders, but there's at least one that drops a whole bunch of Ender pearls, so that's a bonus. There's usually a mine shaft that spawns nearish-by the portal, and I've just farmed spiders from the spawners in there.

**Comment by DrasticDemise** [comment]
Any specific killing quests you would like to see a quest for? I don't really want to make a quest for every single lycanite mob, just the ones that spawn often.

**Comment by Mechalith** [comment]
The chupacabra seem to be the most frequent monster I see. Possibly an afrit would be good. There's also a big grey-green thing that looks like Violator and I can't remember the name of it, but there's a fair number of them, so that'd be good too.

**Comment by DrasticDemise** [comment]
I might just flat out remove the lava monsters if lycanites lobbers do the same thing

**Comment by Mechalith** [comment]
They pretty much fill the same ecological niche, yeah.

**Comment by ConfigsPlease** [comment]
I agree with Mecha, I've not had much of a chance to play lately because of work, but the new Lycanite mobs seem to overrun everything. There's a lot of afrit, as well as those flying water-looking elemental dudes that spawn near one of the lava lakes when I step through the portal, reducing the spawn amounts available elsewhere, for regular mobs. Disabling some of the lava monsters, as well as adding quests (chupacabras, maybe one of the flying-type, like afrit, and possibly one other one for something that's quite difficult to kill, or shadow monsters, i dunno) would make it much more balanced, imo.

As for things being hard to kill - I had something called an Umber Lobber (I think) spawn within 5 minutes in the pool near the portal. It had 150 hearts and for some reason was just firing its projectiles at a wall, which made a bit of lag (when I killed it, my tps went up to normal again). Perhaps reduce the spawn of the ultra-big creatures? I've also had two turtle things, not sure what they're called though, as I killed them via range.

**Comment by Mechalith** [comment]
The trouble with adjusting the spawn weight it that Lycanite's configs don't allow you to specify a subspecies, it just has a setting for how many '\* Lobbers' you can potentially have. The Umber Lobber is supposed to be quite rare, but for whatever reason they REALLY like spawning in that pool.

**Comment by DrasticDemise** [comment]
Vanilla mobs are back to spawning after I changed the configs that out-weighted vanilla

**Comment by Turmfalke2** [comment]
Is there anyway to rescale the size of the monsters, so that it matches their hit box? It is always a bit confusing when you have parts of monsters clipping through the walls.

**Comment by Mechalith** [comment]
That's a Lycanite issue Turmfalke2. It isn't something DD has control over.

**Comment by DrasticDemise** [comment]
I can set mobs to always be the same size, that may provide clarity.

**Comment by Mechalith** [comment]
I haven't checked 0.10 yet but 0.9 has the celphignis set so that natural and forced despawn are both turned off in the Lycanite's config. This appears to ensure that they aren't allowed to despawn at all.

**Comment by DrasticDemise** [comment]
Interesting, I don't think I changed that. Very strange for a default.

**Comment by ConfigsPlease** [comment]
I think the Dark Power quests introduced in v0.9 (or maybe v0.10, which is what I'm running...) for the Lycanite mobs are very well done. Additionally, the spawn rate for vanilla mobs seems to be quite good now - except for zombies. Have you had a look at them, or are they intended to be lower? I've only found one on this world.

**Comment by ConfigsPlease** [comment]
I just noticed that /cofh killall doesn't kill Lycanites mobs - I have about 15 of them hiding under obsidian that naturally formed close to my portal, and they wont despawn. It's half flying mobs too - they just dont have the pathfinding AI to get out. Any ideas on a killall command for Lycanites as well?

**Comment by DrasticDemise** [comment]
/cofh killall \*   should delete everything, but it may be an issue with them not despawning if that doesn't work.

**Comment by ConfigsPlease** [comment]
Adding in the \* works, thanks.

**Comment by Mechalith** [comment]
Celphignis despawning is still disabled in 0.10, just had a chance to check.

**Comment by DrasticDemise** [comment]
Noted. Any other mobs?

**Comment by Mechalith** [comment]
According to a quick search in Notepad++: Yeti, Joust, Joust Alpha, Arisaur, Silex, Concapede, Yale, Maka, Maka Alpha, Ika, Chupacabra, Shade, Aspid.

**Comment by Mechalith** [comment]
There's also four instances of the Lycanite's mobs being set to edit the list of vanilla mob spawns being set to true: freshwater, plains, saltwater, and swamp.

**Comment by DrasticDemise** [comment]
Hmm. Do you encounter hordes of these mobs when the despawning is off?

**Comment by ConfigsPlease** [comment]
The only ones I really have faced so far are the Celp-fish things, Afrit, Lobbers, some turtle-looking thing, no idea what the hell it is, Chupacabra, Shades and Jengus. Everything else only seems to spawn when the 'events' trigger, and even then it's not that many of them. The Celps seem to always have a large number under obsidian, because they have an awful pathfinding it'd seem, or just don't have any set AI.

As for chupacabra/shade, I never have seen more than 1 or 2 at a time, even though my base is right beside a large (VERY large) ravine that I constantly climb up/down to hunt for the damn things.

**Comment by DrasticDemise** [comment]
Do you find that there still aren't enough mobs?

**Comment by ConfigsPlease** [comment]
Well, I have torches covering my main pathways, which may  explain it - I'm not sure if the spawn range for Lycanites is the exact same. As for the "common" ones like Chupacabras, shades, zombies and ESPECIALLY creepers (which I haven't actually found a single one of, either on the minimap or seen on screen... nor have I seen a /killall command list one), I think there could be more. It seems as though the spawn conditions for the 'special' Lycanites mobs - ie., the ones that fly, spawn in water, and whatnot are so much easier to fulfill than the normal ones, that the 'special' ones get spawned a hell of a lot more.

tl;dr I don't think there isn't enough - I think that it needs more weighting towards the ground ones though. After all, it IS a caveworld - why the hell is everything I'm fighting flying into the walls constantly? :P

**Comment by Mechalith** [comment]
I still haven't seen any creepers either, all the spiders have been from a dungeon spawner, and every zombie and skeleton I've seen appear to have spawned from gravestones or the Black Plague ("The dead at your dooooor") event.

**Comment by DrasticDemise** [comment]
I readded the zombie script that I removed earlier so you should see some more. I have also added in wisps to spawn too for the Ethereal Quest. I have also added in a creeper script too.

**Comment by ConfigsPlease** [comment]
Oddly enough, I have seen Skeletons appear inside my base once or twice, which is entirely lit (I've gone over it with f7, and my lighting overlaps quite a lot...). Is there a timer/random event trigger that simply causes a pair of SpecialMobs skeletons to spawn randomly? (Also noted - I only EVER find skeletons in duos, same with Spiders, although most of my spiders are Warded spiders for some damn reason).

**Comment by DrasticDemise** [comment]
That is definitely strange. It may be epic siege mod(which is now removed) that is messing with you.

**Comment by FranklinShack** [comment]
Quick question, what the hell is a lunar grue and what does it do?

Also, I've noticed that once you get to Magic Beam, it becomes incredibly easy to camp by a lava pool and farm Afrits for the dark power quests.

**Comment by ConfigsPlease** [comment]
It's easy to do even with just Rock blast, or whatever it is. I find Magic beam to be iffy on its hitboxes - or at least it was prior to the mobs being all the same size, as I've been busy for a few days and unable to test.

**Comment by DrasticDemise** [comment]
Hmm. The Afrits are the flying bats yeah?

**Comment by FranklinShack** [comment]
Yeah

**Comment by ConfigsPlease** [comment]
I don't know if it's changed in the most recent updates, but I found it wasn't quite as easy to farm them (last played 0.0.10). The reason being that the damn things didn't spawn quite as often - usually, they would only spawn in extremely hard to reach places, whilst all of the 'normal' spawn points that I hadn't lit would be skeletons, chupacabra, etc.

**Comment by DrasticDemise** [comment]
So would the solution here be to remove the Afrit quest or set it on a higher cool down?

**Comment by FranklinShack** [comment]
If say a higher cool down, or to lower the dark power gain, as the normal
mobs are still extremely hard to find. The only Zombies I saw during about
two hits of play were two I spawned in to kill for the seed quest

**Comment by DrasticDemise** [comment]
I still do not understand what is causing vanilla mobs to vanish. There should be no reason that they are so rare, yet I cannot find the correct configs.

Edit: I have found a way to weight around vanilla mobs and have cut the lycanites weight in half.

**Comment by Mechalith** [comment]
I actually just nailed this down. It's the Afrit and Lobbers. They'll both spawn _under_ lava, not just on the surface, which means there's a gigantic open space for them to spawn in that dwarfs everything else. I did some testing and if I disable the Afrit and Lobbers then there's zombies and creepers and skeletons everywhere. They appear to just be maxxing out the limit on entities.

**Comment by DrasticDemise** [comment]
That would actually explain a lot. I will see what I can do about those without having to disable them.
